docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3.7"
 
 services:
-  sqs:
-    image: vsouza/sqs-local
+  redis:
```

Comment:
Nice! We were not even using that container, right?

Reply:
I don’t think so, but I might use for integration testing ...